packages/contracts-bedrock/src/universal/StandardBridge.sol

```diff
@@ -40,10 +38,13 @@ abstract contract StandardBridge {
     /// @notice Mapping that stores deposits for a given pair of local and remote tokens.
     mapping(address => mapping(address => uint256)) public deposits;
 
+    /// @notice Messenger contract on this domain.
+    CrossDomainMessenger internal _MESSENGER;
```

Comment:
Should this have a `@custom:network-specific` natspec tag?

Reply:
Addressed